src/Main/AnimalList/Animal/Animal.js

```diff
@@ -0,0 +1,11 @@
+import './Animal.css';
+
+export default function Animal({ animal }) {
+  return (
+    <div style={{ top: `${animal.top}`, left: `${animal.left}` }} className='animal-card'>
```

Comment:
if you remove the template literals inside of your style, it should position the animals correctly on the page